src/IconButton/index.jsx

```diff
@@ -75,7 +75,7 @@ IconButton.propTypes = {
   /** Click handler for the button */
   onClick: PropTypes.func.isRequired,
   /** Type of button (uses Bootstrap options) */
-  variant: PropTypes.oneOf(['primary', 'secondary', 'success', 'warning', 'danger', 'light', 'dark']),
+  variant: PropTypes.oneOf(['primary', 'secondary', 'success', 'warning', 'danger', 'light', 'dark', 'black']),
```

Comment:
This should be added to the IconButton examples.
(inverted should probably as well...)

Reply:
Update: is now included in 2 of 3 areas for IconButton doc page where "all" variants are listed o.0

Reply:
👍  The black inverse variant was included, but wasn't easy to identify. Fixed the inverse variants section to list each of the individual Icon Buttons for clarity.

Reply:
XD